showcase/src/Badges.elm

```diff
@@ -9,12 +9,16 @@ import Utils exposing (goToDocsCallToAction, prettifyElmCode, story)
 stories cfg =
     storiesOf
         "Badges"
-        [ oneBadge cfg Badge.light "light"
-        , oneBadge cfg Badge.dark "dark"
-        , oneBadge cfg Badge.primary "primary"
-        , oneBadge cfg Badge.warning "warning"
-        , oneBadge cfg Badge.danger "danger"
-        , oneBadge cfg Badge.success "success"
+        [ oneBadge cfg Badge.grayLight "Gray Light"
```

Comment:
Hey, maybe we can merge by color, like having the primaryLight and primaryDark together in the same story.